### PR TITLE
Ch6-issue133

### LINKED ISSUE #133: Edit data split strategies
**Section Outline**
Please list the key points of content you intend to cover for this section
- Edit temporal split section to cover cross validation and “champion” model training in temporal data split.
- Edit cross-validation and nested cross-validation section to cover “champion” model training.
 
**Definition of Done**
- [ ] Edited `5/data-splits.md`
- [ ] Verified against [self-review checklist](https://github.com/aisingapore/ai-practitioner-handbook/blob/main/REVIEWING.md)
- [ ] Added section to [table of contents](https://github.com/aisingapore/ai-practitioner-handbook/blob/main/_toc.yml)
- [ ] Page builds and displays correctly on local machine

### PR DESCRIPTION
**Pull request by joshxinjie**

### CODE REVIEW

**Comment by ngkh2018** [comment]
Good coverage on the data split strategies. Apart from the few typo errors to be corrected, I have no further comments.
- Under 'Nested cross-validation'
   - seperate --> separate
   - evalute --> evaluate
   - paramater --> parameter
   - intially --> initially

File: book/5-data-mgmt-exp-proc/data-splits.md [hunk 1]
```diff
@@ -95,6 +107,20 @@ In scenarios where there are high correlations between successive times, such as
 
 For seasonal data, you may want to take seasonality into account when performing temporal splits. As an example, you can place the first 20 days of every month into the training set, the next 5 days into the validation set, and the last 5 days in the test set.
```
Comment (kevinosl) [inline book/5-data-mgmt-exp-proc/data-splits.md:108]: Do you mean avoid placing February 2 in the training set and February 1 in the validation set to minimise data leakage?

**Review by kevinosl — COMMENTED** [review]
The additions are great Xin Jie.
I did notice something that I have a question about, inline below.
Other than that, all good.

File: book/5-data-mgmt-exp-proc/data-splits.md (hunk 1 above)
Comment (joshxinjie) [reply book/5-data-mgmt-exp-proc/data-splits.md:108]: You are correct, I made a typo here. I will make the edits.

**Review by najibninaba — APPROVED** [review]
Agreed with @kevinosl; the article is good to go.
